test/vectorUtils.test.ts

```diff
@@ -38,13 +41,17 @@ describe('vectorUtils', () => {
   });
 
   describe('offsetPoints', () => {
-    let points: Point[];
+    let points: number[][];
     let normals: Vector2[];
     let newPoints: Point[];
 
     beforeEach(() => {
       const vector = new Vector2(-0.70710678118, 0.70710678118);
```

Comment:
One idea to make the tests more readable is to name the points and/or vectors. Then we could compare without looking at a 12 digit number.